src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,57 @@
+
+<!-- Css for dropdown menu on w3schools -->
+<div class="dropdown">
+    <button class="dropbtn">Settings</button>
+    <div class="dropdown-content">
+      <a href="#" class="btn btn-link">Board Settings</a>
+      <a href="#" class="btn btn-link">Invite</a>
+    </div>
+</div> 
+
+
+<div id="boardSettingsModal" class="modal">
```

Comment:
You keep using the word modal throughout, but this page can be static, it is a separate page that has the board settings and the functionality for inviting someone

Reply:
It has like 3 settings, modals are fine, it does the same thing.